modules/ROOT/pages/appliance/certificates.adoc

```diff
@@ -30,3 +30,5 @@ If you want to limit the access to your server exclusively to HTTPS, use this co
 ....
 ucr set apache2/force_https=yes
 ....
+
+For further information please visit our partner site at https://help.univention.com/t/using-your-own-ssl-certificates/38[Univention]:
```

Comment:
s/:/./g

Reply:
there are 2 ":" in this line. I assume you want the ones at the end to be a full stop, right?